server.go

```diff
@@ -2946,7 +2946,7 @@ func (s *server) peerConnected(conn net.Conn, connReq *connmgr.ConnReq,
 	// htlcs, an extra block is added to prevent the channel from being
 	// closed when the htlc is outstanding and a new block comes in.
 	pCfg := peer.Config{
-		Conn:                    conn,
+		Conn:                    conn.(*brontide.Conn),
```

Comment:
can just use `Conn: brontideConn`

Reply:
There are two other places `*brontide.Conn` is used but I think for now it's ok to leave them and have a follow-up to tackle them.